docs/toolkits/openstack-planning-guide.md

```diff
@@ -1,11 +1,124 @@
+| computing    | Nova           | Virtual computers                       | EC2            |
+| networking   | Neutron        | Software-defined network resources      | VPC            |
+
+Some other commonly-adopted services include:
+
+| Service type            | OpenStack name | What it delivers to users               | AWS Equivalent     |
+|-------------------------|----------------|-----------------------------------------|--------------------|
+| dashboard               | Horizon        | A web interface for using OpenStack     | Management Console |
+| block storage           | Cinder         | Virtual disk drives                     | EBS                |
+| object storage          | Swift          | Data storage buckets                    | S3                 |
+| filesystem              | Manila         | Virtual shared folders                  | EFS                |
+| bare metal              | Ironic         | Real, physical computers                | EC2 Bare Metal     |
+| DNS                     | Designate      | DNS host records                        | Route 53           |
+| orchestration           | Heat           | Automated control of OpenStack services | CloudFormation     |
+| container orchestration | Magnum         | Push-button Kubernetes clusters         | EKS                |
+
+
 ### Control Plane and Data Plane
 
+In a large distributed system like an OpenStack cloud, there is a logical separation between the Control Plane and Data Plane. The control plane consists of the set of OpenStack services (and their dependencies) that _manage_ users' resources on the cloud. The data plane consists of the users' resources themselves. The data plane includes virtual computers, networks, and various types of data storage.
+
+First, let's look at the control plane. It's a sub-set of the diagram at the top of this page.
+
+[![OpenStack Diagram](../img/openstack-diagram-control-plane-only.jpeg)](../img/openstack-diagram-control-plane-only.jpeg)
+
+An OpenStack control plane has a few layers. At the bottom is at least one physical computer (a.k.a. server) to run the control plane services.
+
+The next layer is infrastructure services. These are not part of OpenStack per se, but OpenStack services depend on them to deliver a functioning cloud.
+
+| Service type      | Name(s)                                         | What it does                                              |
+|-------------------|-------------------------------------------------|-----------------------------------------------------------|
+| Database          | MariaDB or MySQL (optionally in Galera cluster) | Stores the persistent state of the control plane          |
+| Message broker    | RabbitMQ                                        | Delivers messages between service components              |
+| API load balancer | HAProxy                                         | Routes and secures network requests to OpenStack services |
```

Comment:
Do we want to say that HAProxy is depended upon by OpenStack services?

Reply:
I believe I do just above
> These are not part of OpenStack per se, but OpenStack services depend on them to deliver a functioning cloud
will clarify a bit further